examples/adaptive_library/BPMTriangularRotorNotches.py

```diff
@@ -50,6 +50,8 @@
 # Import ``pymotorcad`` to access Motor-CAD. Import ``triangular_notch`` to create the notch
 # geometry region with Adaptive Template geometry. Import ``os``, ``shutil``, ``sys``, and
 # ``tempfile`` to open and save a temporary .mot file if none is open.
```

Comment:
```suggestion
# geometry region with the Adaptive Template geometry. Import the ``os``, ``shutil``, ``sys``, and
# ``tempfile`` packages to open and save a temporary MOT file if none is open.
```
Is "Adaptive Template" a proper noun? If it isn't, it should be capitalized. Extensions for files are formatted in all capital letters with no dot.